frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -517,19 +518,32 @@ export const NamespacedClusterServiceVersionList: React.SFC<ClusterServiceVersio
   );
   const NoDataEmptyMsg = () => <MsgBox title="No Operators Found" detail={noDataDetail} />;
 
-  const isCopiedCSV = (source, kind) => {
+  const isCopiedCSV = (source: ClusterServiceVersionKind | SubscriptionKind, kind: string) => {
     return (
       referenceForModel(ClusterServiceVersionModel) === kind && source.status?.reason === 'Copied'
     );
   };
 
-  const removeCopiedCSVs = (operators: (ClusterServiceVersionKind | SubscriptionKind)[]) => {
+  const isStandardCSV = (operator: ClusterServiceVersionKind | SubscriptionKind) => {
```

Comment:
This should always be a CSV and not a Subscription, right?

```suggestion
  const isStandardCSV = (operator: ClusterServiceVersionKind) => {
```

Reply:
is should.. but then the func needs to be call with `isStandardCSV(source as ClusterServiceVersionKind)` since the the source is from `(ClusterServiceVersionKind | SubscriptionKind)[]`